Products/CMFPlone/profiles/default/metadata.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <metadata>
-  <version>5019</version>
+  <version>5020</version>
```

Comment:
@b4oshany Oh, sorry. I noticed that you actually added that ugprade step into plone.app.upgrade... so, probably this is good after all, we just need to get plone.app.upgrade changes merged at first to make tests for this pull pass.

@gforcada How should we properly merge this? Tests are failing, because upgrade step is not yet merged into plone.app.upgrade. Is it correct that we should first merge step into plone.app.upgrade and then this should pass?